com.ibm.wala.cast.python.test/data/tf2_test_callbacks.py

```diff
@@ -0,0 +1,12 @@
+import tensorflow as tf
```

Comment:
What are we testing here? Which API? What is the link to the API? Add answers to these as comments in the files, please.

Reply:
Added.